### ISSUE TITLE
Rule proposal: no skipped tests

### ISSUE THREAD

**Issue by treythomas123** [lo1tuma/eslint-plugin-mocha#54]
It would be nice to have a rule that can warn/fail when skipped tests are found.

There's an [issue](https://github.com/mochajs/mocha/issues/1529) discussing adding something like this to mocha itself, but I would personally prefer to have it in eslint.

I can work on a PR for this if you're open to adding the feature.

**Comment by lo1tuma** [comment]
As I understand the mocha issue, it is only about exclusive tests which we already have rule for.

That’s being I still think it is useful to have a rule to disallow skipped/pending tests. I think this issue is a duplicate of #51. So a Pull Request would be highly welcomed 😉 .

**Comment by treythomas123** [comment]
Wrote [PR #55](https://github.com/lo1tuma/eslint-plugin-mocha/pull/55)!

This partially implements [#51](https://github.com/lo1tuma/eslint-plugin-mocha/issues/51), but not fully because it doesn't catch pending tests, such as: 
```
it('does a thing')
```
It only finds tests that are explicitly skipped.

**Comment by lo1tuma** [comment]
Fixed in #55 and released with 2.1.0
